### PR TITLE
Material Design: Menu

### PR DESCRIPTION
**Pull request by bori00**
I added a menu to the application using the TabBar component from Material Design.

- the menu enables the user to navigate through the 5 pages of the website
- it contains some icons taken from Google Fonts
- the active tab is highlighted

Rather for demonstrational purposes, but I set up some colors for the theme of the website. 

Currently: the website looks like this:

image

### CODE REVIEW

File: src/main/webapp/menu.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    content.setAttribute("class", "mdc-tab__content");
+    content.appendChild(this.createIcon(index));
+    content.appendChild(this.createLabel(index));
+    return content;
+  }
+
+  createTabIndicator(focused) {
+    var indicator = document.createElement("span");
+    if (focused) {
+      indicator.setAttribute("class", "mdc-tab-indicator mdc-tab-indicator--active")
```
Comment (denniswillie) [inline src/main/webapp/menu.js]: maybe change to `indicator.classList.add("mdc-tab-indicator", "mdc-tab-indicator--active")`

File: src/main/webapp/menu.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+  createTabRipple() {
+    var ripple = document.createElement("span");
+    ripple.setAttribute("class", "mdc-tab__ripple");
+    return ripple;
+  }
+
+  createIndicatorContent() {
+    var indicatorContent = document.createElement("span");
+    indicatorContent.setAttribute("class", "mdc-tab-indicator__content mdc-tab-indicator__content--underline");
```
Comment (denniswillie) [inline src/main/webapp/menu.js]: this too.

**Review by denniswillie — COMMENTED** [review]
LGTM!

File: src/main/webapp/menu.js (hunk 1 above)
Comment (bori00) [reply src/main/webapp/menu.js]: Done.

File: src/main/webapp/menu.js (hunk 2 above)
Comment (bori00) [reply src/main/webapp/menu.js]: Done.

**Review by cgollner — DISMISSED** [review]

**Review by cgollner — APPROVED** [review]

**Pull request merged**